draft-ietf-rats-corim.md

```diff
@@ -456,9 +490,26 @@ The following describes each child item of this group.
 ~~~
 
 
-# CoMID {#sec-comid}
+# Concise Module Indentifier (CoMID) {#sec-comid}
+
+A CoMID tag contains information about hardware or firmware component or module to provision
+into a verifier. Each CoMID has a unique identifier that is used to identify tag instances unambigously with in the Verifier.
+
+A CoMID defines several types of Claims, using triples. Each triple defines a specific type of Claim, as given below.
+
+* Reference Triples contain Claims for Reference Values that are expected to match the attested Evidence.
+
+* Endorsed Triples contain Claims for Endorsed Values. These are assertions about the module that do not require evidence matching to be accepted by the Verifier. Specific example includes testing or certification data pertaining to a module, that reflect immutable and additional properties of the module that assists verifiers to derive more information about the attester.
```

Comment:
break into multiple lines.
Change "Specific example includes" to "Specific examples include"
Change "immutable and additional" to "immutable and dynamic"
Change "assits verifiers to derive" to "assist Verifiers to infer"
Change "attester" to "Attester"